src/SourceConstraints.hs

```diff
@@ -59,7 +58,7 @@ runSourceConstraints _options ModSummary{ms_location = ModLocation{..}} parsedMo
 
   pure parsedModule
   where
-    allowLocation = maybe False (not . elem ".stack-work/" . splitPath)
+    allowLocation = maybe False (notElem ".stack-work/" . splitPath)
```

Comment:
@mbj I think this hunk should be extracted to a prefixed `Refactor`.

Reply:
No, because its only possible to do this AFTER we brought `notElem` into scope, via removing the explicit imports.

Reply:
Well, I am aware of that. I was suggesting to first import it and make the refactoring, then have that commit be an atomic switch to no explicit import lists.

Reply:
This is intermediary diff for no reason.